VERSION

```diff
@@ -1 +1 @@
-0.8.3-SNAPSHOT
+dev-master
```

Comment:
Why not 0.9.1-SNAPSHOT?

Reply:
I do not see the point of guessing the next version number.

Reply:
Me neither, actually. Where did dev-master come from?

Reply:
It was random - it sounds cool 😄